app/src/main/res/values-night/colors.xml

```diff
@@ -3,118 +3,27 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <resources xmlns:tools="http://schemas.android.com/tools">
+    <color name="snackbarBackground">#2E255D</color>
     <color name="colorPoweredBy" tools:ignore="MissingDefaultResource">#271948</color>
+
     <!-- Browser Menu items color palette -->
     <color name="mvp_browser_menu_bg" tools:ignore="MissingDefaultResource">@color/photonInk80</color>
-
-    <color name="colorPrimary">#ff272727</color>
```

Comment:
I imagined the intention of adding these colours is so that we switch over to using the themed colours for each mode?

Wouldn't we default to the normal `values/colors.xml` if we remove the `values-night/colors.xml`?